docs/docsite/rst/reference_appendices/release_and_maintenance.rst

```diff
@@ -80,14 +80,16 @@ This table links to the changelogs for each major Ansible release. These changel
 ==================================      ============================    =========================
 Ansible Community Package Release       Status                          Core version dependency
 ==================================      ============================    =========================
-7.0.0                                   In development (unreleased)     2.14
-`6.x Changelogs`_                       Current                         2.13
+8.0.0                                   In development (unreleased)     2.14
+`7.x Changelogs`_                       Current                         2.14
+`6.x Changelogs`_                       Unmaintained (end of life)      2.13
```

Comment:
This needs to change to say it will go EOL after whatever the next 6.x release is. Check with community WG on the number.